.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+.idea/
```

Comment:
I'll remove this on merge, just FYI: These IDE files should be [ignored globally](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).